### DISCUSSION TITLE
The Start and End time requires

### DISCUSSION THREAD

**Discussion by QuentinG50** [Estia-advanced-programming/pandora-public#88] [Q&A]
Should we consider the start of the take off at 00:00:00 or at the time it was this day (since 1970) ?

**Comment by dhmmasson** [comment]
What do you think is the best ?

**Comment by QuentinG50** [comment]
It is easier to consider it at 00:00:00 but more accurate to consider the actual date (cause it is a black box).

**Comment by WilliamDelamare** [comment]
cf wiki for working with dates (consider your machine timezone...)

**Comment by dhmmasson** [comment]
I let the questions open, easier to find them for the other students
